Update vaadin-date-picker for uniform demos

tomalec opened the pull request:
https://github.com/Starcounter/uniform.css/issues/119

tomalec committed:
Update vaadin-date-picker for uniform demos

https://github.com/Starcounter/uniform.css/issues/119

alshakero reviewed (approved):
Can you please add `bower-list.txt` for future PRs?

tomalec committed:
Add bower list for easier inspection of packages,

as suggested at https://github.com/Starcounter/starcounter.github.io/pull/4#pullrequestreview-128301000

tomalec commented:
Sure.
